src/datasources/asset/components/editors/calibration-forecast/query-builder/CalibrationForecastQueryBuilder.test.tsx

```diff
@@ -50,5 +50,12 @@ describe('CalibrationForecastQueryBuilder', () => {
       expect(conditionsContainer?.length).toBe(1);
       expect(conditionsContainer.item(0)?.textContent).toContain(globalVariableOption.label);
     });
+
+    it('should select a sanitized ModelName in query builder', () => {
+      const { conditionsContainer } = renderElement([workspace], [system], 'ModelName = "<script>alert(\'ModelName\')</script>"');
+
+      expect(conditionsContainer?.length).toBe(1);
+      expect(conditionsContainer.item(0)?.innerHTML).not.toContain('alert(\'ModelName\')');
```

Comment:
Is it possible to have an exact quality here? Like expect(conditionsContainer.item(0)?.innerHTML) == ?

Reply:
Not really. We are comparing HTML elements that are generated via the framework. I would keep as it is . We want to make sure we sanitized any alerts and scrips in the HTML.